ios/Classes/SwiftCallKeepPlugin.swift

```diff
@@ -404,6 +404,8 @@ public class SwiftCallKeepPlugin: NSObject, FlutterPlugin, CXProviderDelegate {
             sendEvent(SwiftCallKeepPlugin.ACTION_CALL_ENDED, call.data.toJSON())
             action.fulfill()
         }
+        if(call === self.outgoingCall) {self.outgoingCall = nil}
+        if(call === self.answerCall) {self.answerCall = nil}
```

Comment:
```suggestion
       [(call, &self.outgoingCall), (call, &self.answerCall)].forEach { if $0.0 === $0.1 { $0.1 = nil } }
```

Reply:
I feel like it's clearer the way it is for such simple logic, but maybe @fwal has another opinion

Reply:
Why not always set both to nil?

Reply:
As we only want to set the ended call to `null`, this is in the end call action handler

Reply:
What happens otherwise? In what scenario do we get an end call handler for an outgoing call while we are still in an incoming call?

Reply:
This could use some docs 🙏🏻

Reply:
I feel like this won't occur where we have both an answered call and an outgoing call, but I don't understand why checking which call has ended to clear that call only would be bad